packages/compiler/src/core/types.ts

```diff
@@ -2889,15 +2889,15 @@ export interface RelatedSourceLocation {
 
 export interface LogSink {
   log(log: ProcessedLog): void;
-  trackAction?<T>(asyncAction: () => Promise<T>, log: ProcessedLog): Promise<T>;
+  trackAction?<T>(asyncAction: () => Promise<T>, log: string, completedLog: string): Promise<T>;
```

Comment:
can you actually make this one `@internal` just because I am not sure about this signature so we can't change it in the future

Reply:
we can remove all this if we don't add the output dir in the top line at the end of the action, do we now what we expect on this scenario? :) I just added this as a proposal going with the design accepted decision, but is in a different commit so I can revert it if needed